node/service/src/relay_chain_selection.rs

```diff
@@ -409,6 +426,12 @@ where
 			}
 		};
 
+		tracing::trace!(
+			target: LOG_TARGET,
+			"Ancestor approval restriction applied: {:?}",
+			&subchain_head
```

Comment:
```suggestion
			?subchain_head,
			"Ancestor approval restriction applied",
```

The nice thing about the tracing log macros is that you can forward this information much better than putting them into "text".